Docker.md

```diff
@@ -0,0 +1,171 @@
+## Docker
+
+[Docker](https://www.docker.com) is a tool that enables packaging software as a [container image](https://www.docker.com/resources/what-container) that includes all of the dependencies to run a given application. In the context of this project, Docker enables building and running the `tbb` application across multiple operating systems.
+
+### Installation
+
+To get started, follow the official [Getting Started](https://docs.docker.com/get-docker/) guide to install Docker for your operating system. If you are using Linux, be sure to also follow [the guide for installing docker-compose](https://docs.docker.com/compose/install/#install-compose-on-linux-systems). This extra step is not necessary on Windows and MacOS.
+
+Once all tools are installed, you will have two new executables that you can run in your terminal:
+1. `docker` - This is the application used for building and running containers
+2. `docker-compose` - [Docker Compose](https://docs.docker.com/compose/) is a tool for defining how to run any number of containers
```

Comment:
Starting with the review.

OMG! You added a well written intro to Docker and even how to set it up for the students. Well formatted. Easy to read with all correct links. You made me really smile. Thanks for your amazing contribution!